builder/vmware/vmx/step_clone_vmx.go

```diff
@@ -22,22 +22,27 @@ func (s *StepCloneVMX) Run(_ context.Context, state multistep.StateBag) multiste
 	driver := state.Get("driver").(vmwcommon.Driver)
 	ui := state.Get("ui").(packer.Ui)
 
+	// initially we need to stash the path to the original .vmx file
 	vmxPath := filepath.Join(s.OutputDir, s.VMName+".vmx")
 
+	// so first, let's clone the source path to the vmxPath
 	ui.Say("Cloning source VM...")
 	log.Printf("Cloning from: %s", s.Path)
 	log.Printf("Cloning to: %s", vmxPath)
 	if err := driver.Clone(vmxPath, s.Path); err != nil {
 		state.Put("error", err)
 		return multistep.ActionHalt
 	}
+	ui.Say(fmt.Sprintf("Successfully cloned source VM to: %s", vmxPath))
```

Comment:
I think this is probably better as a debug statement (log.Printf) rather than a ui.Say statement.

Reply:
K. I'll fix it.

Reply:
I'll just remove it because the log.Printf on line 31 outputs the vmxPath to the log anyways. As I mentioned in my latest comment, it's emitted so that users can see the successful state of calling out to the driver to clone the source .vmx as it's the only function that depends on executing an external program.